server/model/article.js

```diff
@@ -38,7 +38,7 @@ export class ArticleFactory {
 
     const mainImage: Picture = getWpFeaturedImage(json.featured_image, json.attachments);
     const wpThumbnail = json.post_thumbnail;
-    const thumbnail: Picture = wpThumbnail ? {
+    const thumbnail: ?Picture = wpThumbnail ? {
```

Comment:
It makes it an Option / Maybe.
It's wither a `Picture` or `null` in JS land.

https://flowtype.org/docs/nullable-types.html